src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -175,7 +175,10 @@ trait BackendCall {
       else
         Future.successful(Option(dockerImage))
 
-    call.task.runtimeAttributes.docker map hashDockerImage getOrElse Future.successful(None) map hashGivenDockerHash
+    if (workflowDescriptor.configCallCaching)
```

Comment:
Ok if it's too annoying don't bother, I was just wondering if it would avoid having the same `if`s everywhere we do `getHash` but it's not that bad.